cmd/restic/global.go

```diff
@@ -498,7 +498,7 @@ func OpenRepository(ctx context.Context, opts GlobalOptions) (*repository.Reposi
 			id = id[:8]
 		}
 		if !opts.JSON {
-			Verbosef("repository %v opened (repository version %v) successfully, password is correct\n", id, s.Config().Version)
+			Verbosef("repository %v opened (repository version %v, compression level %v)\n", id, s.Config().Version, opts.Compression.String())
```

Comment:
I think that it would be better if we ditch the second "repository" word such that the text is just this instead: `repository %v opened (version %v, compression level %v)\n`

It's so utterly obvious that the first version mentioned refers to the repository that was opened, and if there are any other types of versions to name in the future they can be explicitly mentioned, e.g. with `repository %v opened (version %v, compression level %v, determination version %v, metadata version %v)\n`.

Reply:
Is changed.